frontend/src/views/dataset/CreateDataset.vue

```diff
@@ -88,7 +93,12 @@ export default {
 						console.log(response)
 					}
 				})
-			}) 
+			}).catch((response) => {
+				console.log("HERES ThE CATCH")
```

Comment:
"ThE" typo

Reply:
Agh - debug typo - this catch shouldn't be in here.

Reply:
Added commit to remove this.